agents/mpath/fence_mpath.py

```diff
@@ -332,6 +332,7 @@ def main():
 		fail_usage("Failed: No devices found")
 
 	options["devices"] = [d for d in re.split(r"\s*,\s*|\s+", options["--devices"].strip()) if d]
+	options["--plug"] = re.sub(r'^0+', '', re.sub(r"^0x", "", options["--plug"]))
```

Comment:
You can shorten this to the inner function, e.g. `r"^0x0*"` for 0 or more matches of the trailing 0 or more or `r"^0x0?"` for 0 or 1 match of it.

Reply:
Hi @oalbrigt 

Since we need to take the following cases into account, we have changed the conversion to an or condition.

```
multipath.conf - reservation_key 01
pcmk_host_map - "nodeA:01"
```
How do you think?

Best Regards,
Hideo Yamauchi.

Reply:
Looks good to me.

Reply:
Oh. It should be 0x0* if not it matches "x" 0 or more times.

Reply:
Hi @oalbrigt 

> Oh. It should be 0x0* if not it matches "x" 0 or more times.

Sorry .. I made a mistake.
I pushed again.

Many thanks,
Hideo Yamauchi.